### ISSUE TITLE
[Extension] Tutorial for Google analytics is not properly update. 

### ISSUE THREAD

**Issue by AzrizHaziq** [GoogleChrome/developer.chrome.com#2062]
I'm migrating my manifest 2 to 3 and face few difficulty with google analytic
Since manifest version 3 migrate to service worker and it doesn't allow any third party script anymore. 

Even tho I explicitly state in my manifest.json `content_security_policy.extension_pages` to have `script-src 'self' https://ssl.google-analytics.com` it will show error below. I did try a lot of combination with `https://www.google-analytics.com` and non works.

![image](https://user-images.githubusercontent.com/9023187/151914540-a54c874d-1b91-4530-843a-77129c1be07b.png)

Web: https://developer.chrome.com/docs/extensions/mv3/tut_analytics/
GH: https://github.com/GoogleChrome/developer.chrome.com//blob/main/site/en/docs/extensions/mv3/tut_analytics/index.md

this is my full csp in manifest.json
```json
  "content_security_policy": {
    "extension_pages": "default-src 'self'; style-src 'self'; object-src 'self';  script-src https://www.google-analytics.com/analytics.js 'self'"
 }
  ```

I do found other issue in chrome extension group:  https://groups.google.com/a/chromium.org/g/chromium-extensions/c/0mPEDNJA0XM/m/vfaOPTykAAAJ
Hence, what would I expect is a working steps to setup google analytics in chrome manifest version 3.

**Desktop (please complete the following information):**
 - OS: macOS 11.6.2 (20G314)
 - Browser: Chrome Version 97.0.4692.99 (Official Build) (x86_64)

**Comment by lsat-extension** [comment]
+1 Also would like to use Google Analytics to track my extension!

**Comment by AzrizHaziq** [comment]
Hi any update on this 🙂.

**Comment by nikhiltri** [comment]
Looking for an update here as well. Manifest 3 is now required to post a new version of our app. Would like guidance as we rely on the recommendations in the tutorial to add this functionality to our extension.

**Comment by tarasinf** [comment]
+1,
* one idea was to use firebase/analytics, unfortunately it [doesn't support extensions](https://firebase.google.com/docs/web/environments-js-sdk)

**Comment by praveensbabu** [comment]
Facing the same issue, can't add the Google analytics on Chrome extension manifest v3
Issues : 
1. When using https://www.google-analytics.com/analytics.js, Manifest v3 doesn't support external resources
2. When downloading the analytics.js file and adding it, shows another error

> `Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval'". Either the 'unsafe-inline' keyword, a hash ('sha256-EHyOnMkEOB8pNSL3baDzuqKTbGusqfCCk0DtZlLVpKs='), or a nonce ('nonce-...') is required to enable inline execution.`

Note am using the analytics script

> `(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments);},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m);
        })(window,document,'script','./analytics.js','ga');`

**Comment by AmySteam** [comment]
@oliverdunk @IanStanion-google I thought you might want to know about this when updating these docs to be MV3 compatible.

**Comment by AmySteam** [comment]
Fixed in #6619
